libr/anal/esil.c

```diff
@@ -424,6 +430,18 @@ R_API bool r_anal_esil_pushnum(RAnalEsil *esil, ut64 num) {
 	return r_anal_esil_push (esil, str);
 }
 
+R_API bool r_anal_esil_pushbig(RAnalEsil *esil, ut8 *num, ut32 size) {
```

Comment:
looks unused and wrong

Reply:
This is going to be used for float80, float128 and larger multiplication results, I am working on it now.

Reply:
I decided to leave this to another PR. So this PR will only support 32 and 64 bit floats.